specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -259,14 +333,28 @@ model CertificateMetadata {
   name?: string;
 }
 
-@doc("The load test configuration.")
+@added(APIVersions.v2023_04_01_preview)
+@doc("Auto stop criteria for a test.")
+model AutoStopCriteria {
+  @doc("Auto stop is enabled or not")
+  autoStopEnabled?: boolean;
+
+  @doc("Value of auto stop criteria metric")
```

Comment:
update description to specific threshold percentage of errors when test run should be automatically stopped